scipy/stats/_morestats.py

```diff
@@ -3030,7 +3031,7 @@ def bartlett(*samples, axis=0):
 LeveneResult = namedtuple('LeveneResult', ('statistic', 'pvalue'))
 
 
-@xp_capabilities(np_only=True)
+@xp_capabilities()
```

Comment:
Torch GPU fails in CI because it doesn't have `fdtrc`. I imagine JAX will fail, too.
```suggestion
@xp_capabilities(cpu_only=True, exceptions=['cupy'])
```
I still find this a bit indirect - I would prefer to declare that this fails with Torch and JAX on GPU rather than make exceptions to `cpu_only=True`. But @lucascolley, you mentioned during our SDG call that you think that making exceptions is the preferred way?